torchvision/datasets/vision.py

```diff
@@ -12,7 +12,7 @@ class VisionDataset(data.Dataset):
     It is necessary to override the ``__getitem__`` and ``__len__`` method.
 
     Args:
-        root (string): Root directory of dataset.
+        root (string, optional): Root directory of dataset.
```

Comment:
While we're at it, it might be worth clarify that this parameter is not critical anyway

```suggestion
        root (string, optional): Root directory of dataset. Only used for `__repr__`.
```